modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -10197,8 +10197,9 @@ export const console: ConsoleNS = {
                                 label: "Assigned application",
                                 placeholder: "Select application to assign the role",
                                 applicationSubTitle: {
-                                    application: "Support application-scoped roles. ",
-                                    organization: "Support organization-scoped roles"
+                                    application: "Support application-scoped roles.",
+                                    organization: "Support organization-scoped role. ",
+                                    changeAudience: "Change the audience"
```

Comment:
```suggestion
                                    changeAudience: "Change the audience."
```

Reply:
This is a link text which does not require `.`. WDYT?